crates/ef-testing/src/evm_sequencer/sequencer/mod.rs

```diff
@@ -197,3 +200,87 @@ pub fn convert_contract_class_v1(class: &CompiledClass) -> Result<ContractClass,
         casm_contract_class,
     )?))
 }
+
+lazy_static! {
+    pub static ref INITIAL_SEQUENCER_STATE: SequencerState = {
+        let mut state = SequencerState::default();
+
+        // On KakarotZero we rely on the Cairo1 helpers class for unavailable syscalls and precompiles.
+        #[cfg(feature = "v0")]
+        let storage = vec![
+            (OWNABLE_OWNER, *KAKAROT_OWNER_ADDRESS.0.key()),
+            (KAKAROT_NATIVE_TOKEN_ADDRESS, *ETH_FEE_TOKEN_ADDRESS.0.key()),
+            (KAKAROT_ACCOUNT_CONTRACT_CLASS_HASH, ACCOUNT_CONTRACT_CLASS_HASH.0),
+            (KAKAROT_CAIRO1_HELPERS_CLASS_HASH, CAIRO1_HELPERS_CLASS_HASH.0),
+            (KAKAROT_BLOCK_GAS_LIMIT, Felt::from(BLOCK_GAS_LIMIT)),
+            (KAKAROT_UNINITIALIZED_ACCOUNT_CLASS_HASH, UNINITIALIZED_ACCOUNT_CLASS_HASH.0),
+        ];
+
+        #[cfg(not(feature = "v0"))]
+        let storage = vec![
+            (OWNABLE_OWNER, *KAKAROT_OWNER_ADDRESS.0.key()),
+            (KAKAROT_NATIVE_TOKEN_ADDRESS, *ETH_FEE_TOKEN_ADDRESS.0.key()),
+            (KAKAROT_ACCOUNT_CONTRACT_CLASS_HASH, ACCOUNT_CONTRACT_CLASS_HASH.0),
+            (KAKAROT_BLOCK_GAS_LIMIT, Felt::from(BLOCK_GAS_LIMIT)),
+            (KAKAROT_UNINITIALIZED_ACCOUNT_CLASS_HASH, UNINITIALIZED_ACCOUNT_CLASS_HASH.0),
+        ];
+
+        // Write all the storage vars to the sequencer state.
+        for (k, v) in storage {
+            (&mut state).set_storage_at(*KAKAROT_ADDRESS, get_storage_var_address(k, &[]), v).expect("failed to set storage");
+        }
+
+        // Convert the contract classes to using the util corresponding to the Cairo version it's compiled with.
+        let (converted_kakarot_class, converted_account_class, converted_uninitialized_class) = {
+            #[cfg(not(feature = "v1"))]
```

Comment:
use `v0` and remove the negation ?

Reply:
causes an issue with clippy about the expected types for "default" targets - if I only have v0 and v1 target, the expected type is a (), whereas here it understands that in _all_ cases is returned a tuple of size 3

Reply:
maybe add a third case like the rest of the code base ?
Shouldn't it panics if not V0 or V1 ?

Reply:
I don't think I can because this must be done at compile time, not runtime here because of type differences